test-data/everest/egg/everest/model/config_flow.yml

```diff
@@ -1,5 +1,5 @@
 definitions:
-  eclbase: eclipse/model/EGG
+  realization_folder: r{{configpath}}/../../eclipse/include/realizations/realization-r{{realization}}/eclipse
```

Comment:
Not needed? Can just remove the whole `definitions` section?

Reply:
Actually might make sense to keep `eclbase` here as it is being reused later in the config.